doc/source/index.rst

```diff
@@ -263,6 +264,67 @@ An example script simulating a simple cut is below:
   % Plot
   plot(cut_sim);
 
+
+Performance and Memory Tips
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following are a few tips to help make sure you have the optimum settings
+for the type of work you're doing and your computing resources.
+
+**Number of Threads**
+
+Euphonic will make use of the C Extension by default, and automatically choose
+the number of threads using the Python function ``multiprocessing.cpu_count``.
+However, this can be overridden by the ``use_c`` and ``n_threads`` arguments
+to ``CoherentCrystal``. Ensure that if these arguments are used, they are
+appropriate to the computing resource you are using. Generally ``use_c``
+should be  ``true`` and ``n_threads`` should be the same as the number of
+cores.
+
+**Chunking**
+
+The phonon eigenvectors, which are an intermediate step in calculating
+the scattering intensities, are particularly memory intensive, requiring
+:math:`18n^2` floating point numbers per q-point, where n is the number
+of atoms in the unit cell of your calculation. To reduce memory consumption,
+the intensity calculation can be chunked with the ``chunk`` argument to
+``CoherentCrystal``. This defines the number of q-points that are calculated
+at once. By default the calculation is not chunked at all, so this argument
+may be useful if you find you are running out of memory (which may cause
+mysterious crashes!). It is best to use the largest chunk you can
+get away with based on the amount of memory available and the number of atoms
+in the unit cell.
```

Comment:
Oops! I'll fix that

And yes it would be nice to have a tool to determine the optimum chunk, Python makes sense as it's easier and can also be used by `euphonic_sqw_models` users too, I'll make an issue.